test/operations-loops-3.js

```diff
@@ -0,0 +1,18 @@
+var assert = require("assert");
```

Comment:
completely wrong

here you should require your existing solution
```
 test/operations-conditions-3.js
```

and call function from it in your loop